lib/Backends/BackendUtils.cpp

```diff
@@ -81,6 +74,38 @@ runtime::RuntimeBundle::getSymbolInfo(const Named *v) const {
   return it->second;
 }
 
+/// If \p PH is an output placeholder, \returns true.
+/// This is determined by checking if the PH has a user which uses the PH as an
+/// overwritten input.
+bool checkIfOutput(const Placeholder *PH) {
```

Comment:
Consider renaming this to `isOutput`, `isOutputPH` or `isOutputPlaceholder`. That makes it more explicit that the return value is a boolean.